README.md

```diff
@@ -1 +1,149 @@
+- **export** – Export the json records that make up an entity for debugging or reingesting after tweaks are made.
+
+It can also be used to view the reports generated by the following companion projects ...
+- https://github.com/Senzing/g2snapshot
+- https://github.com/Senzing/g2audit
+
+Usage:
+
+```console
+python3 G2Explorer.py  --help 
+usage: G2Explorer.py [-h] [-c INI_FILE_NAME] [-s SNAPSHOT_FILE_NAME]
+                     [-a AUDIT_FILE_NAME] [-D DEBUG_OUTPUT]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -c INI_FILE_NAME, --config_file_name INI_FILE_NAME
+                        name of the g2.ini file, defaults to
+                        /etc/opt/senzing/G2Module.ini
+  -s SNAPSHOT_FILE_NAME, --snapshot_json_file SNAPSHOT_FILE_NAME
+                        the name of a json statistics file computed by
+                        G2Snapshot.py
+  -a AUDIT_FILE_NAME, --audit_json_file AUDIT_FILE_NAME
+                        the name of a json statistics file computed by
+                        G2Audit.py
+  -D DEBUG_OUTPUT, --debug_output DEBUG_OUTPUT
+                        print raw api json to screen or <filename.txt>
+```
+
+## Contents
+
+1. [Prerequisites](#Prerequisites)
+2. [Installation](#Installation)
+3. [Typical use](#Typical-use)
+4. [Exploring data](#Exploring-data)
+
+### Prerequisites
+- python 3.6 or higher
+- Senzing API version 2.4 or higher
+- python pretty table module (pip3 install ptable)
+
+### Installation
+
+1. Simply place the the following files in a directory of your choice ...  
+    - [G2Explorer.py](G2Explorer.py) 
+
+2. Set PYTHONPATH environment variable to python directory where you installed Senzing.
+    - Example: export PYTHONPATH=/opt/senzing/g2/python
+
+3. The senzing environment must be set to your project by sourceing the setupEnv script created for it.
+
+Its a good idea to place these settings in your .bashrc file to make sure the enviroment is always setup and ready to go.
+*These will already be set if you are using a Senzing docker image such as the sshd or console.*
+
+### Typical use
+```console
+python3 G2Explorer.py 
+```
+Optional parameters ...
+- The -c config_file parameter is only required if your project's G2Module.ini file can't be found in the usual location.
+- The -s snapshot_file_name parameter is for convenience if you just took a snapshot and want to immediately view it.
+- The -a audit_file_name parameter is for convenience if you just ran an audit and want to immediately view it.
+- The -d debug_output parameter turns on debug to either screen or to a file name.  This shows you what the API is returning 
+as you search and get records.  
+-- use -d SCREEN to print the debug output to the screen
+-- or -d /path/to/filename.txt to print the debug output to whatever file you choose.
+
+### Exploring data
+
+The best documentation for how to use the G2Explorer and its companion projects is located here ...
```

Comment:
Further documentation for using G2Explorer and it's companion utilities can be located at: